test/strategies/in-memory-storage.spec.js

```diff
@@ -10,9 +10,9 @@ describe('in-memory-strategy', () => {
     );
   });
 
-  it('should throw if no namespace in setItem', () => {
+  it('should throw if no namespace in setItem', async () => {
     const capsule = new InMemoryStorageCapsule();
-    expect(() => capsule.setItem('shahata', 123)).to.throw(
+    await expect(capsule.setItem('shahata', 123)).to.eventually.rejectedWith(
```

Comment:
Why is this change needed? `setItem` is now defined as `async`, but actually, it contains no `await` and doesn't return a promise, so it is not really `async`, and this code doesn't need to change.

Reply:
Nevermind, I now realized that `async` changes what `setItem` returns, and require changing the test.